pkg/sql/parser/sql.y

```diff
@@ -5226,6 +5226,12 @@ show_roles_stmt:
   }
 | SHOW ROLES error // SHOW HELP: SHOW ROLES
 
+// %Help: SHOW ZONE CONFIGURATION - display current zone configuration
+// %Category: Cfg
+// %Text: SHOW ZONE CONFIGURATION FOR [ RANGE | DATABASE | TABLE | INDEX ] <name>
+// SHOW ZONE CONFIGURATION FOR PARTITION OF [ INDEX | TABLE ] <name>
```

Comment:
Please replace `FOR` with `FROM`.

Reply:
@andyyang890 , But , I am seeing `FOR` in the docs - https://www.cockroachlabs.com/docs/stable/show-zone-configurations.html

Reply:
It was changed in https://github.com/cockroachdb/cockroach/pull/59410

Reply:
done, thanks